### PR TITLE
add: change language in login

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Refactor change language method
- Add html in login
- TODO add the final styles

### CODE REVIEW

File: src/components/Login/Login.js [hunk 1]
```diff
@@ -80,15 +80,36 @@ class Login extends React.Component {
...
-                {this._('login.signup')}
-              </Link>
-            </small>
+            <div className="language-selector">
+              {this.intl.locale === 'es' ? (
+                <div className="option" id="spanish-selector">
+                  <span>
+                    <strong>ES</strong>
+                  </span>
+                  <Link to="/login?lang=en" className="link-green uppercase">
```
Comment (andresmoschini) [inline src/components/Login/Login.js]: Is it not possible making it agnostic to current page to allow us a more easily refactoring?

For example something like:

```suggestion
                  <Link to="?lang=en" className="link-green uppercase">
```

Comment (andresmoschini) [inline src/components/Login/Login.js]: `link-green` class is still necessary?

```suggestion
                  <Link to="/login?lang=en" className="uppercase">
```

File: src/i18n/DopplerIntlProvider.js [hunk 2]
```diff
@@ -14,7 +14,11 @@ const messages = {
 addLocaleData([...en, ...es]);
 
 export default ({ locale, children }) => (
-  <IntlProvider locale={locale || 'en'} messages={flattenMessages(messages[locale || 'en'])}>
+  <IntlProvider
+    key={locale}
```
Comment (andresmoschini) [inline src/i18n/DopplerIntlProvider.js]:
```suggestion
    key={locale || 'en'}
```

Like in `locale` property?

File: src/components/Login/Login.js [hunk 3]
```diff
@@ -80,15 +80,36 @@ class Login extends React.Component {
         <article className="main-panel">
           <header>
             <h1 className="logo-doppler-new">Doppler</h1>
-            <small className="content-signin">
-              {this._('login.you_want_create_account')}{' '}
-              <Link to="/signup" className="link-green uppercase">
-                {this._('login.signup')}
-              </Link>
-            </small>
+            <div className="language-selector">
```
Comment (andresmoschini) [inline src/components/Login/Login.js]: What do you think about also apply it in Signup page and also refactoring it as a component?

Comment (fcoronelmakingsense) [reply src/components/Login/Login.js]: you talk always small tasks and small changes 🤣  but i can

Comment (andresmoschini) [reply src/components/Login/Login.js]: You can do it in another PR, no problem.

Just take into account that the task of "select language in non-authenticated pages" will be only completed if all non-authenticated pages have this option.

File: src/i18n/DopplerIntlProvider.js (hunk 2 above)
Comment (fcoronelmakingsense) [reply src/i18n/DopplerIntlProvider.js]: fixed

File: src/components/Login/Login.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply src/components/Login/Login.js]: fixed

Comment (fcoronelmakingsense) [reply src/components/Login/Login.js]: fixed

Comment (andresmoschini) [reply src/components/Login/Login.js]: Please, test it, I am not sure.

Comment (andresmoschini) [reply src/components/Login/Login.js]: Wow, and it works?

Comment (fcoronelmakingsense) [reply src/components/Login/Login.js]: yes!!

Comment (fcoronelmakingsense) [reply src/components/Login/Login.js]: yes it's works!

**Pull request merged**